src/localization/messages_pt_BR.js

```diff
@@ -3,8 +3,9 @@
  * Locale: PT (Portuguese; português)
  * Region: BR (Brazil)
  */
-$.extend( $.validator.messages, {
-	required: "Este campo &eacute; obrigat&oacute;rio.",
+$.extend($.validator.messages, {
```

Comment:
Please add one space after `(` and one before `)` in all lines in this file.
See http://contribute.jquery.org/style-guide/js/#spacing